hw_2/HW2_S_Glenn.r

```diff
@@ -84,10 +87,12 @@ head(vipcls$del_date)
 vipcls$del_date[grep("-1--1--1", vipcls$del_date)] <- NA
 vipcls$del_date <- as.Date(vipcls$del_date)
 ##c. days between enrollment and delivery
-enrollToDelDays <- difftime(vipcls$del_date, vipcls$enroll_date)
+vipcls$enrollToDelDays <- difftime(vipcls$del_date, vipcls$enroll_date)
 ##d. Investigate which ptid has a delivery date prior to enrollment
 ptid <- vipcls$patid[vipcls$enrollToDelDays < 0]
+ptid <- ptid[!is.na(ptid)]
 ##e. Set implausible number of days between enrollment and delivery to missing
+vipcls$enrollToDelDays[vipcls$enrollToDelDays < 0] <- NA  
```

Comment:
Here, when the number of days between enrollment and delivery is less than 0, we set that value to NA.